libr/util/assert.c

```diff
@@ -0,0 +1,25 @@
+#include <r_util.h>
+
+#define S_SIZE 200
+
+R_API void r_log_warn(const char *file, int line, const char *func, const char *warnexpr) {
+	char s[S_SIZE];
+
+	s[0] = '\0';
+	if (warnexpr) {
+		snprintf (s, S_SIZE, "(%s:%d):%s%s runtime check failed: (%s)\n",
+			  file, line, func, func[0] ? ":" : "", warnexpr);
+	} else {
+		snprintf (s, S_SIZE, "(%s:%d):%s%s code should not be reached\n",
+			  file, line, func, func[0] ? ":" : "");
+	}
+	eprintf ("WARNING: %s", s);
```

Comment:
why not just eprintf instead of snprintf and then eprintf?

Reply:
ok